src/nat/finetuning/interfaces/trajectory_builder.py

```
@@ -0,0 +1,115 @@
+        Finalize the trajectory building process and return the constructed trajectories.
+
+        Args:
+            run_id (str): The unique identifier for the training run.
+            meta (dict): Metadata associated with the training run.
+
+        Returns:
+            list[Trajectory]: The list of constructed trajectories.
+        """
+        raise NotImplementedError
+
+    async def compute_reward(self, output_item: EvalOutputItem, meta: dict | None = None):
```

Ruff (0.14.8)

Unused method argument: `meta`

(ARG002)